stl/inc/format

```diff
@@ -611,17 +696,35 @@ _NODISCARD constexpr const _CharT* _Parse_replacement_field(
     return _Begin + 1;
 }
 
+template <class _CharT>
+const _CharT* _Scan_format_string(const _CharT* _First, const _CharT* _Last, const _CharT _Val, const _Cvtvec& _Cvt) {
+    if (_Cvt._Mbcurmax == 1 || _Cvt._Mbcurmax == 4) {
+        return _Find_unchecked(_First, _Last, _Val);
+    }
+
+    while (_First != _Last && *_First != _Val) {
+        const int _Units = _Code_units_in_next_character(_First, _Last, _Cvt);
+        if (_Units < 0) {
+            throw format_error("Invalid encoded character in format string.");
+        }
+        _First += _Units;
+    }
+
+    return _First;
+}
+
 template <class _CharT, _Parse_replacement_field_callbacks<_CharT> _HandlerT>
-constexpr void _Parse_format_string(basic_string_view<_CharT> _Format_str, _HandlerT&& _Handler) {
-    auto _Begin = _Format_str.data();
-    auto _End   = _Begin + _Format_str.size();
+void _Parse_format_string(basic_string_view<_CharT> _Format_str, _HandlerT&& _Handler) {
+    auto _Begin         = _Format_str.data();
+    auto _End           = _Begin + _Format_str.size();
+    const _Cvtvec& _Cvt = ::_Getcvt();

```

Comment:
All of the `_Getcvt` member functions are gone, so fully-qualifying `::_Getcvt` is probably no longer necessary even for clarity.